be/src/exprs/vectorized/runtime_filter.h

```diff
@@ -336,13 +337,51 @@ class RuntimeBloomFilter final : public JoinRuntimeFilter {
     }
 
     size_t evaluate(Column* input_column, RunningContext* ctx) const override {
-        if (_hash_partition_number != 0) {
+        if (_num_hash_partitions != 0) {
             return t_evaluate<true>(input_column, ctx);
         } else {
             return t_evaluate<false>(input_column, ctx);
         }
     }
 
+    void compute_hash_values_for_multi_part(RunningContext* running_ctx, int8_t join_mode, const Column* input_column,
+                                            std::vector<uint32_t>& hash_values, size_t num_rows) const {
+        typedef void (Column::*HashFuncType)(uint32_t*, uint32_t, uint32_t) const;
+
+        auto compute_hash = [&input_column, &num_rows, &hash_values, this](HashFuncType hash_func,
+                                                                           size_t num_hash_partitions) {
+            hash_values.assign(num_rows, HashUtil::FNV_SEED);
```

Comment:
Init value of `fnv_hash` is `HashUtil::FNV_SEED` , but init value of `crc32_hash` is 0.

Reply:
yes